dashboard/src/pages/storage/aips/[id]/index.vue

```diff
@@ -48,6 +49,7 @@ onMounted(() => {
       </div>
       <div class="col-md-6">
         <AipLocationCard />
+        <AipReportsCard v-if="aipStore.isDeleted" />
```

Comment:
Doesn't hurt to check if the AIP is deleted here, but I think the checks in the component are enough.